airbyte-cdk/bulk/toolkits/extract-cdc/src/main/kotlin/io/airbyte/cdk/read/cdc/CdcPartitionReader.kt

```diff
@@ -150,7 +153,9 @@ class CdcPartitionReader<T : Comparable<T>>(
             // At this point, if we haven't returned already, we need to close down the engine.
             log.info { "Shutting down Debezium engine: ${closeReason.message}." }
             // TODO : send close analytics message
-            Thread({ engine.close() }, "debezium-close").start()
+            runBlocking() {
+                launch(Dispatchers.IO + Job()) { engine.close() }
+            }
```

Comment:
No functional change here.
Doing the same thing using coroutines

(debezium requires the the engine is shut down from a different thread)